sonoff/sonoff_template.h

```diff
@@ -1102,14 +1102,17 @@ const mytmplt kModules[MAXMODULE] PROGMEM = {
      GPIO_REL1_INV,    // GPIO16 Green Led 1 (0 = On, 1 = Off)
   },
   { "OBI Socket",      // OBI socket (ESP8266) - https://www.obi.de/hausfunksteuerung/wifi-stecker-schuko/p/2291706
-     0, 0, 0, 0,
-     GPIO_LED1,        // GPIO04 LED on top and in switch button
-     GPIO_REL1,        // GPIO05 Relay 1 (0 = Off, 1 = On)
+     GPIO_USER,        // GPIO00
+     0,0,0,
+     GPIO_LED1,        // GPIO04 Blue LED
+     GPIO_REL1,        // GPIO05 (Relay OFF, but used as Relay Switch)
      0, 0, 0, 0, 0, 0, // Flash connection
-     GPIO_LED2,        // GPIO12
-     0,                // GPIO13
-     GPIO_KEY1,        // GPIO14 switch button
-     0, 0, 0
+     GPIO_LED4,        // GPIO12 (Relay ON, but set to LOW, so we can switch with GPIO05)
```

Comment:
Sorry, stills not understand why this?

Reply:
Because it isn't really used as LED on this device (as mentioned in the comment)
It's just set to LED so it stays LOW.
and this way LED2 can be used with the user-configurable GPIOs

Reply:
I thought it might be more confusing, when adding an LED, that you can't choose LED2 even though there isn't an LED2

Reply:
Sorry, not convinced.

I think that the actual template is easier to understand.

If you want to control the relay led, may be you can set that GPIO as another GPIO_USER

Reply:
and let the user to select what to do with that.

What do you think?

Reply:
you can't controll the relay-led.
GPIO_12 must be set to LOW so the relay can be switched with GPIO_5
There is only one controllable LED which is at GPIO_4 (Blue led)
But I can change back to LED2 if you think that is better.